main.py

```diff
@@ -8,11 +8,11 @@
 
 ## SETTINGS
 # How many seconds to wait in-between songs
-seconds_between_songs = 10
+seconds_between_songs = int(os.environ.get('BUSTY_COOLDOWN_SECS', 10))
```

Comment:
```suggestion
seconds_between_songs = int(os.environ.get("BUSTY_COOLDOWN_SECS", 10))
```

For consistency with the lines below.